config/default_sonar_profile.xml

```diff
@@ -9,45 +9,36 @@ e.g. - provided link corresponds to rule S3776
   <name>checkstyle-profile</name>
   <language>java</language>
   <rules>
-    <!-- till #4912 -->
-    <!--
     <rule>
       <repositoryKey>common-java</repositoryKey>
       <key>DuplicatedBlocks</key>
       <priority>MAJOR</priority>
       <parameters/>
     </rule>
-    -->
```

Comment:
@romani I also not so sure about these uncommented sonar items. Are they needed?

Reply:
we do not use this file , we do not use sonar in docker - https://github.com/checkstyle/checkstyle/issues/4912

Reply:
done https://github.com/checkstyle/checkstyle/pull/9284